README.md

```diff
@@ -126,10 +126,11 @@ npm run dev
         ],
 ```
 
-**3. Update token expiry. Default: 3600 seconds (1 hr)**
+**3. Update token expiry. Default: 28800 seconds (8 hrs)**
+- Default expiry set to 8 hrs in order to reduce CPU workload for auth requests. Check [discussion](https://github.com/ably/laravel-broadcaster/issues/55#issuecomment-2582845145).
 - Set `ABLY_TOKEN_EXPIRY` in **.env** file.
 ```dotenv
-    ABLY_TOKEN_EXPIRY=21600
+    ABLY_TOKEN_EXPIRY=43200
```

Comment:
If this is seconds, then this number is 12 hours.

Reply:
Yeah, this is intentional. 
Keeping timeout low causes CPU consumption because of number of auth requests.
For each channel, client sends an auth request for authorization. If a client has 30 channels and timeout is 1 hr, every 1 hour, each client will make 30 separate requests. So, aim is to reduce requests by 8 times or 12 times by making timeout more than given value

Reply:
This will surely need long term fix ( updating token before expiry ), but currently we don't have capacity to do the same.

Reply:
Reason to increase expiry can be found by issue raised by one of the customer -> https://github.com/ably/laravel-broadcaster/issues/55

Reply:
Yeah, acutally, we have already default value as 8 hours, so here it's mentioned differently.